kwamp-client-core/src/main/kotlin/com/laurencegarmstrong/kwamp/client/core/Client.kt

```diff
@@ -85,6 +87,7 @@ class ClientImpl(
         }
     }
 
+
```

Comment:
Most other function just have a single newline between them, so I don't think we need one here?